lib/TimezoneGuesser/FindFromOutlookCities.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabre\VObject\TimezoneGuesser;
+
+use DateTimeZone;
+
+class FindFromOutlookCities implements TimezoneFinder
+{
+    /**
+     * Example: TZID:(UTC+01:00) Bruxelles\, København\, Madrid\, Paris
+     */
+    public function find(string $tzid, bool $failIfUncertain = false): ?DateTimeZone
+    {
+        preg_match('/TZID:\(UTC(\+|\-)\d{2}:\d{2}\)/', $tzid, $matches);
+
+        if (empty($matches)) {
+            return null;
+        }
+
+        // Remove TZID:(UTC+01:00)
+        $tzid = trim(substr($tzid, strlen($matches[0])));
```

Comment:
Would use preg_replace to remove it instead

Reply:
I want to just return when the pattern is not match, and don't need to continue the following codes

Reply:
Sorry, I don't understand... I meant using preg_replace to remove the TZID:(UTC+00:00) part from the string, so that you have only the list of cities

Reply:
I found I can use the $count variable in the preg_replace to know if the pattern matches

Reply:
I like!